src/GitReportCL/RunProcess.cs

```diff
@@ -0,0 +1,105 @@
+﻿
+
+namespace GitReportCL
+{
+    using System;
+    using System.Diagnostics;
+    using System.IO;
+
+    class RunProcess
+    {
+        public Validation validate = new Validation();
+        private string SetDirectory()
```

Comment:
Method name of `SetDirectory` have couple problems.

* Usually developer create methods `SetSomething` in order to set field called `Something`. 
* Usually methods `SetSomething` have value given by argument
* Usually methods `SetSomething` does not return anything.